modules/op-viewing-pipeline-logs-in-kibana.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+// cicd/pipelines/viewing-pipeline-logs-using-the-openshift-logging-operator.adoc
+//
+
+[id="op-viewing-pipeline-logs-in-kibana_{context}"]
+= Viewing pipeline logs in Kibana
+
+To view pipeline logs in the Kibana web console:
+
+.Procedure
+
+. Log in to {product-title} web console as a cluster administrator.
+.. In the top right of the menu bar, click the *grid* icon → *Observability* → *Logging*. The Kibana web console is displayed.
+
+. Create an index pattern:
+.. On the left navigation panel of the *Kibana* web console, click *Management*.
+.. Click *Create index pattern*.
+.. Under *Step 1 of 2: Define index pattern* → *Index pattern*, enter a *`pass:[*]`* pattern and click *Next Step*.
+.. Under *Step 2 of 2: Configure settings* → *Time filter field name*, select *@timestamp* from the drop-down menu, and click *Create index pattern*.
+
+. Add a filter:
+.. On the left navigation panel of the *Kibana* web console, click *Discover*.
+.. Click *Add a filter +* → *Edit Query DSL*.
++
+[NOTE]
+====
+* For each of the example filters that follows, edit the query and click *Save*.
+* The filters are applied one after another.
+====
++
+... Filter the containers related to pipelines:
++
+.Example query to filter pipelines containers
+[source,json]
+----
+{
+  "query": {
+	"match": {
+  	"kubernetes.flat_labels": {
+    	"query": "app_kubernetes_io/managed-by=tekton-pipelines",
+    	"type": "phrase"
+  	}
+	}
+  }
+}
+----
++
+... Filter all containers that are not `place-tools` container. As an illustration of using the graphical drop-down menus instead of editing the query DSL, consider the following approach:
++
+.Example of filtering using the drop-down fields
+image::../../images/not-placetools.png[Not place-tools]
++
+... Filter `pipelinerun` in labels for highlighting:
++
+.Example query to filter `pipelinerun` in labels for highlighting
+[source,json]
+----
+{
+  "query": {
+	"match": {
+  	"kubernetes.flat_labels": {
+    	"query": "tekton_dev/pipelineRun=",
+    	"type": "phrase"
+  	}
+	}
+  }
+}
+----
++
+... Filter `pipeline` in labels for highlighting:
```

Comment:
I think the `+` before each of these `...` steps is causing it to render all as one step?
e.g. "Filter `pipeline` in labels for highlighting:" shows in the preview as part of the above "Filter `pipelinerun` in labels for highlighting:" step